src/v/cluster/plugin_frontend.cc

```diff
@@ -238,17 +238,23 @@ errc plugin_frontend::validate_mutation(const transform_cmd& cmd) {
     for (const auto& topic : noproduce) {
         no_sink_topics.emplace(topic);
     }
-    validator v(_topics, _table, std::move(no_sink_topics));
+    const size_t max_transforms
+      = config::shard_local_cfg().wasm_per_core_memory_reservation.value()
+        / config::shard_local_cfg().wasm_per_function_memory_limit.value();
```

Comment:
safe to assume this is gtz at this point?

Reply:
Yeah there is a validation for the config that the min is 64KiB. Good thing to ask tho!